addons/point_of_sale/models/pos_config.py

```diff
@@ -459,7 +367,12 @@ def create(self, vals_list):
         # If you plan to add something after this, use a new environment. The one above is no longer valid after the modules install.
         return pos_configs
 
+    def _set_default_on_vals(self, vals):
+        if 'tip_product_id' in vals and not vals['tip_product_id']:
+            vals['tip_product_id'] = self.env.ref('point_of_sale.product_product_tip', False).id
```

Comment:
Why `raise_if_not_found=False`? That case is not handled so if the xid is not found, this is just going to break with a less clear error.